terraform/examples/alicloud-drds-instance/main.tf

```diff
@@ -0,0 +1,14 @@
+provider "alicloud" {
```

Comment:
I think the example had better show a VPC drds instance.

Reply:
Done.